package.json

```diff
@@ -35,13 +35,14 @@
     "underscore.string": "^3.2.2"
   },
   "devDependencies": {
-    "broccoli-plugin": "^1.2.0",
+    "broccoli-plugin": "rwjblue/broccoli-plugin#add-capabilities",
```

Comment:
Note: update this to 1.3.0 when https://github.com/broccolijs/broccoli-plugin/pull/19 is landed and released.